packages/bundle/src/adaptiveCards/Attachment/AdaptiveCardRenderer.js

```diff
@@ -84,19 +84,19 @@ function addEventListenerOnceWithUndo(element, name, handler) {
   return detach;
 }
 
+function addEventListenerWithUndo(element, name, handler) {
+  element.addEventListener(name, handler);
+
+  return () => element.removeEventListener(name, handler);
+}
+
 function disableElementWithUndo(element) {
   const undoStack = [];
   const isActive = element === document.activeElement;
   const tag = element.nodeName.toLowerCase();
 
   /* eslint-disable-next-line default-case */
   switch (tag) {
-    // Should we not disable <a>? Will some of the <a> are styled as button?
-    case 'a':
```

Comment:
should we also  remove 'a' on line 134 ?

Reply:
yeah, thanks for the catch